apps/pano/tsconfig.json

```diff
@@ -4,7 +4,18 @@
     "baseUrl": ".",
     "paths": {
       "~/*": ["./src/*"]
-    }
+    },
+    "allowJs": true,
```

Comment:
Aren't these rules getting extended by "@kampus/tsconfig/remix.json"?

Reply:
upgrading to remix auto generated all of these, these will be addressed in the kampus-ui pr.

Reply:
💯